webpack.config.js

```diff
@@ -198,24 +198,32 @@ module.exports = {
           canPrint: true,
         })
       : noop,
-    ...MAIN_PACKAGES.map(
-      template =>
-        new HtmlWebpackPlugin({
-          filename: `${template}.html`,
-          template: join(`src/${template}/${template}.html`),
-          chunks: [template],
-          minify: prd
-            ? false
-            : {
-                collapseWhitespace: true,
-                removeComments: true,
-                removeRedundantAttributes: true,
-                removeScriptTypeAttributes: true,
-                removeStyleLinkTypeAttributes: true,
-                useShortDoctype: true,
-              },
-        })
-    ),
+    ...MAIN_PACKAGES.map(template => {
+      const usePlatformSpecific = fs.existsSync(
+        join(`src/${template}/${template}.${platform}.html`)
```

Comment:
awesome. maybe this should only apply to "background" package?

Reply:
I wanted to, initially, but I left it, in case we ever, in future, need to have separate packages for firefox and chrome. Wdyt?

Reply:
oh https://github.com/Teamweek/browser-extension/blob/master/webpack.config.js#L64

ideally, we don't duplicate the other 2 for now...

Reply:
I guess I'd just restrict it to background.

Reply:
Done!

Reply:
Awesome